### PR TITLE
Search results view

### PR DESCRIPTION
**Pull request by OBrandon**
Created the styling for the search results. 
image

### CODE REVIEW

File: app/templates/org/view_profile.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h3>Gallery</h3>
   </div>
 </section>
 <section style="padding-top: 10vh">
-          {% include 'partials/_footer.html'%}
+          {% include 'partials/_secondary_footer.html'%}
```
Comment (stevenbursztyn) [inline app/templates/org/view_profile.html]: what's the difference between these 2 footers?

Comment (OBrandon) [reply app/templates/org/view_profile.html]: yea its based off the mockups so footer has the header Ready to connect while secondary_footer does not.

Comment (stevenbursztyn) [reply app/templates/org/view_profile.html]: oh i see. let's leave the regular footer in and we'll make changes later if needed 😀

File: app/assets/styles/app.scss [hunk 2]
```diff
@@ -15,6 +15,16 @@ p {
   z-index: 2000;
 }
 
+// img cropping for search
+.crop {
+    width: 300px;
```
Comment (riyer15) [inline app/assets/styles/app.scss]: change to 200px by 200px

File: app/assets/styles/app.scss [hunk 3]
```diff
@@ -15,6 +15,16 @@ p {
...
+    height: 300px;
+    overflow: hidden;
+}
+
+.crop img {
+    height: 325px;
```
Comment (riyer15) [inline app/assets/styles/app.scss]: change to 225 px

File: app/static/styles/app.css [hunk 4]
```diff
@@ -15,43 +15,56 @@ p {
 }
 
 @media -sass-debug-info{filename{}line{font-family:\0000319}}
+.crop {
```
Comment (riyer15) [inline app/static/styles/app.css:18]: dont change it here! it will be overwritten

File: app/static/styles/app.css [hunk 5]
```diff
@@ -154,33 +167,37 @@ h4 {
...
 }
 
-@media -sass-debug-info{filename{}line{font-family:\00003152}}
+@media -sass-debug-info{filename{}line{font-family:\00003162}}
 section .col-md-6 {
   padding-left: 0px;
   padding-right: 0px;
 }
 
+<<<<<<< HEAD
```
Comment (riyer15) [inline app/static/styles/app.css]: take out the merge conflict code please

**Pull request merged**